src/shared/sandboxRequest.ts

```diff
@@ -21,7 +21,7 @@ export const generateSboxName = async (): Promise<string> => {
 };
 
 export async function createSandboxRequest(
-  isClone: true,
+  isClone: true | boolean,
```

Comment:
This could be false | undefined then it should work.

Reply:
In this case I need the `true` function signature, given the command is asking for a clone of a sandbox. `org create sandbox` does not use the src sandbox name from the return, but clone must be true in order to run the correct code in the function. This code is also used in the force version of the commands, where that code does need the src sandbox name.

I rechecked the types for each call and type script is telling me that each is getting the expected return type. I guess testing will be the let us know.